### PR TITLE
Update changelog for 3.1.0-RC2

### PR DESCRIPTION
**Pull request by KengoTODA**
Now we have no opened issue for Milestone 3.1.0, then let's release RC2 to ask users to have a try.

After we merge this PR, I will make commits like a34d991b9bf877853641b4f79d786f87ee154c26 and release RC2. I also update spotbugs.github.com to provide Eclipse update site (I think I don't need to make PR for this, because https://github.com/spotbugs/spotbugs.github.io/pull/12 has not merged yet).

After that, I will push change like 9751a22ac595fbb0e2989357a361de7346d3eaa9 to switch back version to SNAPSHOT.

@jsotuyod Can I ask you to handle release for Gradle plugin? It seems we haven't released it yet.
https://plugins.gradle.org/search?term=spotbugs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 This is the changelog for SpotBugs. This follows [Keep a Changelog v0.3](http://keepachangelog.com/en/0.3.0/).
 
+## 3.1.0-RC2 (2017/Apr/22)
```
Comment (KengoTODA) [inline CHANGELOG.md]: This date is just a temporal date

**Comment by KengoTODA** [comment]
I will reopen this after we stabilize Eclipse plugin.

**Comment by KengoTODA** [comment]
Fix for Eclipse still needs some time, so I will reopen this to release Gradle plugin and test-harness.

**Comment by jsotuyod** [comment]
@KengoTODA sorry, I've been out of town the last several weeks. I'll gladly do it if you haven't already. Remember I sent you by email the keys needed to be able to publish the plugin yourself.

**Pull request merged**